.github/workflows/container-scan.yml

```diff
@@ -0,0 +1,37 @@
+name: "Scan containers"
+on:
+  schedule:
+    # every night at one
+    - cron:  '0 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  get-containers:
+    name: Get containers from submodules
+    runs-on: ubuntu-latest
+    outputs:
+      container-matrix: ${{ steps.submodules.outputs.containers }}
```

Comment:
Should we be getting latest or latest release here?

Reply:
It takes all releases. The cve-database behind the scan is updated frequently, thus we can find issues in released versions once they are found.

Reply:
I mean the GE code base - `master` is bleeding edge, `releases` is last stable release of the GE code.

Reply:
The versions are taken from the github-releases api, therefore only tests versions that the GE maintainer officially released(pre-releases are excluded). It does not directly work on the git code base. See https://github.com/FIWARE-Ops/get-images-from-submodules#output for some version outputs.

Reply:
NTC